src/Admin/MaintainRoles/components/AddRole.js

```diff
@@ -78,7 +78,7 @@ const AddRole = ({
           )}
           <button
             type="button"
-            className="button margin-left-15 margin-top-large"
+            className="button greyButtonNoMinWidth margin-left-15 margin-top-large"
```

Comment:
Seeing as these grey buttons are cropping up more often now, I think it's probably a good time to move it into a shared component within new-nomis-shared and use styled-components. We can also change the neighbouring green button to use `@govuk-react/button` while we're in there too.  We can also look at adding a `<ButtonContainer>` for when there are side by side buttons.

Have a look at [ButtonCancel.js](https://github.com/ministryofjustice/prisonstaffhub/blob/master/src/Components/PayOtherForm/elements/ButtonCancel.js) in prisonstaffhub. I'll sit with you on Monday and we can do it together.

Besides my comments, I'll merge it anyway as I don't want to hold the ticket up.